src/index.ts

```diff
@@ -172,11 +174,30 @@ class FsspecWidget extends Widget {
     // browserAreaLabel.classList.add('jfss-browseAreaLabel');
     // browserAreaLabel.innerText = 'Browse Filesystem';
     // lowerArea.appendChild(browserAreaLabel);
+    this.lowerAreaHeader = document.createElement('div');
+    this.lowerAreaHeader.classList.add('jfss-lowerAreaHeader');
+    lowerArea.appendChild(this.lowerAreaHeader);
 
     this.selectedFsLabel = document.createElement('div');
     this.selectedFsLabel.classList.add('jfss-selectedFsLabel');
     this.selectedFsLabel.innerText = '<Select a filesystem>';
-    lowerArea.appendChild(this.selectedFsLabel);
+    this.lowerAreaHeader.appendChild(this.selectedFsLabel);
+
+    const spacer = document.createElement('div');
+    spacer.style.flexGrow = '10';
+    spacer.style.width = '1.5rem';
+    spacer.style.height = '1.5rem';
+    this.lowerAreaHeader.appendChild(spacer);
+
+    this.refreshFileList = document.createElement('div');
```

Comment:
Error indicates this button element needs a unique name?

Reply:
Hey Martin, yes, I think this can be updated in the tests, so that we grab the item by a different selector that will resolve to the one refresh button of interest. I will push some updates shortly that grab the item by title so that should hopefully resolve the error!